notifier.py

```diff
@@ -1,32 +1,63 @@
-import telegram
 import logging
 import random
+import webbrowser
+
+import telegram
+
 
 class NullNotifier:
     def notify(self, properties):
         pass
 
+    @staticmethod
+    def get_instance(config):
```

Comment:
moved initialization here, so it can create the default Notifier if we only set "enabled", or the Browsifier if we have enabled=true and type=browser